src/components/PathwayCourse/index.js

```diff
@@ -191,6 +190,20 @@ function PathwayCourse() {
     return item.id == pathwayId;
   });
 
+  let SupplementalCourse;
+  let filterPathwayCourse;
+
+  if (pathwayId == 2) {
+    filterPathwayCourse = pathwayCourse?.data?.courses.filter(
+      (item) => item.name === "Spoken-English"
```

Comment:
Here as well if object item don`t contained name in it app might crash `item?.name` will be better to be used